modules/ROOT/pages/dataweave-type-reuse-java-classes.adoc

```diff
@@ -0,0 +1,70 @@
+= Reusing Types from Java Classes
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the import directive with the `java!` prefix when loading a Java class. The prefix `java!` tells DataWeave to use the Java module loader. When the module is loaded, you can call constructors, variables, and functions (see xref:dataweave-cookbook-java-methods.adoc[]). The class type is also available. DataWeave exposes the type of the Java class with the type name `Class`.
+
+Java classes are mapped to the DataWeave types listed in xref:dataweave-formats-java.adoc#java-value-mapping[Java Value Mapping].
+If a Java class is not present in the table, DataWeave treats the class as a `JavaBean` and maps it to a DataWeave `Object` type. The keys in that object match the names of the properties in the Java class and their values correspond to the DataWeave type that matches the property class. The following example shows how DataWeave takes all properties from the Java getters:
```

Comment:
I wonder about "present". specified? defined? listed? included?

Reply:
```suggestion
If a Java class is not present in the table, DataWeave treats the class as `JavaBean` and maps it to a DataWeave `Object` type. The keys in that object match the names of the properties in the Java class and their values correspond to the DataWeave type that matches the property class. The following example shows how DataWeave takes all properties from the Java getters:
```

Reply:
This is a mouthful - any way to simplify?

> The keys in that object match the names of the properties in the Java class and their values correspond to the DataWeave type that matches the property class.

Reply:
I like the options. Will go with _specified_. 

Indef art deleted.